api/src/main/java/org/pmiops/workbench/reporting/ReportingUploadServiceInsertQueryImpl.java

```diff
@@ -29,12 +30,12 @@
 
 @Service("REPORTING_UPLOAD_SERVICE_DML_IMPL")
```

Comment:
I changed this and then backed it out. Since the annotation is outside the class's scope, so it has to point to a public static method, which means at the injection site you wind up with a reference to the actual class being injected, which is supposed to be abstracted away from you. The string doesn't really belong to the class, but is an external identifier.

I suppose you could make a file with all the qualifiers, but that wouldn't be very legible.